packages/devtools/client/exports/mount.mjs

```diff
@@ -0,0 +1,45 @@
+import { parseQuery } from 'ufo';
```

Comment:
Should we add this module to `source.include`?

DevTools may not work in legacy browsers.

Reply:
okk, I will do it.

Reply:
I refactor mount.mjs in es5. Now it should work well with legacy browsers.

Reply:
Ok, we can add a compile step in the future because maintain es5 code is hard.

Reply:
Sure.